### ISSUE TITLE
How can I change the property displayed?

### ISSUE THREAD

**Issue by ColdZoo** [adadgio/neo4j-js#6]
Hi, 
    neo4j-js is the first GUI graph editor I can find,  great work! 
     My problem is , I managed to connecte to my local neo4j server. But I don't know how to let the graph display the right property. I assume this due to wrong settings.js configuration. In my case:
![image](https://user-images.githubusercontent.com/11594715/32760168-0556f2ea-c928-11e7-92f4-1f3dad436d79.png)
There is no label on the green node. And the 'text' attribute of that node is supposed to show. 
Here is my settings.js:
![image](https://user-images.githubusercontent.com/11594715/32760230-4acc040a-c928-11e7-96db-e85d70bab81b.png)

     Many thanks

**Comment by adadgio** [comment]
Hmm, i looked at this this morning and i think unfortunately that the "name" property is hard coded... This project was quite experimental and i'm working on a Angular2/Typescript version because this one is quite a mess to maintain... ;-).

Let me get back to you about that though...

**Comment by adadgio** [comment]
Yep... If you look at AppBundle/Component/Graph.js, you see this code: 

```
_g.gnodes.append('text')
                .attr('class', 'label')
                .text(function(d) {
                    var idField  = Settings.graph._id;
                    var prop = Settings.graph.primaryLabel;

                    return d._properties.name;
                    // if (typeof(d._properties[prop]) === 'undefined') {
                    //     return d[idField];
                    // } else {
                    //     return d._properties[prop];
                    // }
                });

            _g.force.start();
```

Make me think i hard coded the `name` property. A pull request would be welcome ! :-)

**Comment by adadgio** [comment]
Note: a brand new version is here ! https://github.com/adadgio/neo4j-js-ng2

**Comment by ColdZoo** [comment]
Great, though not quite familiar with JS codes. I will try to fix it . thanks. 

btw, has the new version ok  with that problem?

**Comment by adadgio** [comment]
Absolutely, the new version is focused on fixes first. In your case it lets you configure the properties displayed.
